src/containers/views/tests/datafileedit.spec.js

```diff
@@ -84,8 +84,9 @@ describe('Containers::DataFileEdit', () => {
 
   it('should call deleteDataFile', () => {
     const { deleteButton, actions } = setup();
+    window.confirm = jest.fn(() => true);
```

Comment:
I think we can mock this globally in `tools/fileMock.js`.

Reply:
I have moved the mocks into `tools/fileMock.js` and renamed it into `tools/jestMocks.js`...